modules/lvgl/input/lvgl_pointer_input.c

```diff
@@ -46,15 +54,6 @@ static void lvgl_pointer_process_event(const struct device *dev, struct input_ev
 		return;
 	}
 
-	/* adjust coordinates */
-	if (cfg->swap_xy) {
```

Comment:
This change was included within [https://github.com/zephyrproject-rtos/zephyr/pull/70541](https://github.com/zephyrproject-rtos/zephyr/pull/70541)